<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InputWidget</class>
 <widget class="QWidget" name="InputWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>602</width>
    <height>469</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Угроза «Авария на электростанции»</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Кол-венная информация</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Количество производителей</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="mSpinBox">
        <property name="minimum">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Количество потребителей</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="nSpinBox">
        <property name="minimum">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Количество резервных хранилищ</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QSpinBox" name="tSpinBox">
        <property name="minimum">
         <number>1</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Качественная информация</string>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Объемы производства энергоресурсов</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QToolButton" name="aToolButton">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Объемы потребления энергоресурсов</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QToolButton" name="bToolButton">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Объемы энергоресурсов, поставляемых потребителям </string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QToolButton" name="ixToolButton">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Объемы энергоресурсов в резервных хранилищах</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QToolButton" name="iuToolButton">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
      <item row="10" column="1">
       <widget class="QToolButton" name="ifToolButton">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
      <item row="10" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Приоритеты связей между потребителями и пунктами резервного хранения</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Вероятности аварии у производителей ресурсов</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QToolButton" name="ipToolButton">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>nSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>InputWidget</receiver>
   <slot>onQtyChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>266</x>
     <y>92</y>
    </hint>
    <hint type="destinationlabel">
     <x>702</x>
     <y>92</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>InputWidget</receiver>
   <slot>onQtyChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>262</x>
     <y>51</y>
    </hint>
    <hint type="destinationlabel">
     <x>606</x>
     <y>50</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>InputWidget</receiver>
   <slot>onQtyChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>269</x>
     <y>132</y>
    </hint>
    <hint type="destinationlabel">
     <x>609</x>
     <y>128</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>aToolButton</sender>
   <signal>clicked()</signal>
   <receiver>InputWidget</receiver>
   <slot>onOpenDisplayTableDialogPushed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>563</x>
     <y>206</y>
    </hint>
    <hint type="destinationlabel">
     <x>659</x>
     <y>207</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ipToolButton</sender>
   <signal>clicked()</signal>
   <receiver>InputWidget</receiver>
   <slot>onOpenDisplayTableDialogPushed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>574</x>
     <y>245</y>
    </hint>
    <hint type="destinationlabel">
     <x>700</x>
     <y>254</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bToolButton</sender>
   <signal>clicked()</signal>
   <receiver>InputWidget</receiver>
   <slot>onOpenDisplayTableDialogPushed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>565</x>
     <y>287</y>
    </hint>
    <hint type="destinationlabel">
     <x>684</x>
     <y>287</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ixToolButton</sender>
   <signal>clicked()</signal>
   <receiver>InputWidget</receiver>
   <slot>onOpenDisplayTableDialogPushed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>563</x>
     <y>326</y>
    </hint>
    <hint type="destinationlabel">
     <x>676</x>
     <y>328</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>iuToolButton</sender>
   <signal>clicked()</signal>
   <receiver>InputWidget</receiver>
   <slot>onOpenDisplayTableDialogPushed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>576</x>
     <y>380</y>
    </hint>
    <hint type="destinationlabel">
     <x>716</x>
     <y>384</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ifToolButton</sender>
   <signal>clicked()</signal>
   <receiver>InputWidget</receiver>
   <slot>onOpenDisplayTableDialogPushed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>564</x>
     <y>426</y>
    </hint>
    <hint type="destinationlabel">
     <x>753</x>
     <y>425</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onQtyChanged()</slot>
  <slot>onOpenDisplayTableDialogPushed()</slot>
 </slots>
</ui>
